src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-ProfileCard/Components/index.test.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { mount, render } from 'enzyme';
+import emotionSerializer, { matchers } from 'jest-emotion';
+import { UserProfileCard, IconContent, DropDownButton } from './index';
+import * as S from '../styles';
+import { Props } from '../types';
+import { AppIcon } from '../../../../types/iconsTypes/iconEnums';
+
+const props: Props = {
+  content: 'Send Message',
+  name: AppIcon.QuestionAnswerIcon,
+};
+
+let wrapper;
+
+expect.addSnapshotSerializer(emotionSerializer);
+expect.extend(matchers);
+
+describe('<UserProfileCard />', () => {
+  beforeEach(() => {
+    wrapper = mount(<UserProfileCard />);
```

Comment:
It is used only in second test case. So remove it and replace.

Reply:
Done